File: app/src/main/java/com/booboot/vndbandroid/api/bean/Producer.java
package com.booboot.vndbandroid.api.bean;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/**
 * Created by od on 25/04/2016.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class Producer extends VNDBCommand {
    private int id;
    private boolean developer;
    private boolean publisher;
    private String name;
    private String original;
    private String type;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public boolean isDeveloper() {
        return developer;
    }

    public void setDeveloper(boolean developer) {
        this.developer = developer;
    }

    public boolean isPublisher() {
        return publisher;
    }

    public void setPublisher(boolean publisher) {
        this.publisher = publisher;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getOriginal() {
        return original;
    }

    public void setOriginal(String original) {
        this.original = original;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/main/java/com/booboot/vndbandroid/model/vndb/Producer.kt
package com.booboot.vndbandroid.model.vndb

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Producer(
    var id: Int = 0,
    var developer: Boolean = false,
    var publisher: Boolean = false,
    var name: String = "",
    var language: String = "",
    var original: String? = null,
    var description: String? = null,
    var type: String = ""
)